include/gridtools/meta/fold.hpp

```diff
@@ -55,7 +55,8 @@ namespace gridtools {
             };
             template <template <class...> class F,
                 class S,
-                template <class...> class L,
+                template <class...>
```

Comment:
that's strange. it looks like your `clang-format` behaves  differently then mine. mine has the version `6.0.0`

Reply:
That's strange yes... I have `8.0.0`. I reverted these changes and check this in more detail later.